dist/jsoneditor.js

```diff
@@ -1715,7 +1715,15 @@ JSONEditor.AbstractEditor = Class.extend({
     return null;
   },
   getTitle: function() {
-    return this.schema.title || this.key;
+      //Title collapsable form sections
+      if (this.hasOwnProperty("hide_add_button")){
```

Comment:
Maybe instead of checking for this specific property, just check for parent and parent key, and fallback on its `plural_title` or `title` before `key`